libs/labelbox/src/labelbox/data/serialization/ndjson/converter.py

```diff
@@ -106,14 +106,16 @@ def serialize(
                     if not isinstance(annotation, RelationshipAnnotation):
                         uuid_safe_annotations.append(annotation)
             label.annotations = uuid_safe_annotations
-            for example in NDLabel.from_common([label]):
-                annotation_uuid = getattr(example, "uuid", None)
+            for annotation in NDLabel.from_common([label]):
+                annotation_uuid = getattr(annotation, "uuid", None)
 
-                res = example.dict(
+                res = annotation.dict(
                     by_alias=True,
                     exclude={"uuid"} if annotation_uuid == "None" else None,
                 )
                 for k, v in list(res.items()):
                     if k in IGNORE_IF_NONE and v is None:
                         del res[k]
+                if getattr(label, 'is_benchmark_reference'):
```

Comment:
did you try to declare `class Label(_CamelCaseMixin)` , I do not think you would need this line

Reply:
I tend to avoid juggling with mixins because this caused MRO problems in the past. Do you think we can leave it as is?

Reply:
the way I have it above should prob work, can you try? If it does not work (i.e. test fails etc) we could use your approach, but I rather not have custom serialization if we can avoid it

Reply:
actually, the logic is the following: if there is `is_benchmark_reference` attribute set for the label, then all child annotations should have `isBenchmarkReferenceLabel` set to True. I don't see how the code above would help to avoid this logic